### ISSUE TITLE
Null injector error for translate service

### ISSUE THREAD

**Issue by t-zander** [ghiscoding/Angular-Slickgrid#128]
**I'm submitting a bug report**

* **Angular Version:**
6
major.minor.patch

* **Library Version:**
1.9.5
major.minor.patch

* **Browser:**
all | Chrome XX | Firefox XX | Edge XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView

* **Language:**
TypeScript 2.6

**Current behavior:**
I follwed all instructions in your docs. I installed ngx translate even despite the fact I don't use it at all...
Now I am getting the error

> StaticInjectorError(AppModule)[AngularSlickgridComponent -> TranslateService]: 
> StaticInjectorError(Platform: core)[AngularSlickgridComponent -> TranslateService]: 
> NullInjectorError: No provider for TranslateService!
In module
```
TranslateModule.forRoot(),
AngularSlickgridModule.forRoot(),
```

In child module where I tried to use slickgrid...
```
imports: [
    CommonModule,
    BrowserModule,
    FormsModule,
    ...
    TranslateModule,
    AngularSlickgridModule
  ],
```
My package json
```
"dependencies": {
    "@agm/core": "^1.0.0-beta.3",
    "@agm/js-marker-clusterer": "^1.0.0-beta.5",
    "@angular/animations": "^6.0.0",
    "@angular/cdk": "^6.0.0",
    "@angular/common": "^6.0.0",
    "@angular/compiler": "^6.0.0",
    "@angular/core": "^6.0.0",
    "@angular/forms": "^6.0.0",
    "@angular/http": "^6.0.0",
    "@angular/platform-browser": "^6.0.0",
    "@angular/platform-browser-dynamic": "^6.0.0",
    "@angular/router": "^6.0.0",
    "@ngui/map": "^0.20.2",
    "@ngx-translate/core": "^10.0.2",
    "@ngx-translate/http-loader": "^3.0.0",
   ...
    "angular-slickgrid": "^1.9.5",
   ...
    "rxjs": "^6.0.0",
    "rxjs-compat": "^6.2.1",
    "scrollreveal": "^3.3.6",
    "text-mask-addons": "^3.7.1",
    "web-animations-js": "^2.3.1",
    "zone.js": "0.8.26"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "~0.6.8",
    "@angular/cli": "^6.0.0",
    "@angular/compiler-cli": "^6.0.0",
    "@angular/language-service": "^6.0.0",
    "@types/jasmine": "~2.5.53",
    "@types/jasminewd2": "~2.0.2",
    "@types/node": "~6.0.60",
    "@types/scrollreveal": "0.0.3",
    "codelyzer": "^4.0.1",
    "jasmine-core": "~2.6.2",
    "jasmine-spec-reporter": "~4.1.0",
    "karma": "~1.7.0",
    "karma-chrome-launcher": "~2.1.1",
    "karma-cli": "~1.0.1",
    "karma-coverage-istanbul-reporter": "^1.2.1",
    "karma-jasmine": "~1.1.0",
    "karma-jasmine-html-reporter": "^0.2.2",
    "protractor": "^5.4.0",
    "ts-node": "~3.2.0",
    "tslint": "~5.7.0",
    "typescript": "~2.7.2"
  }
```
What am I doing wrong?
**Expected/desired behavior:**

* **What is the expected behavior?**

* **What is the motivation / use case for changing the behavior?**

**Comment by ghiscoding** [comment]
I actually skipped Angular 6 completely and went directly to 7, so I never tested myself with Angular 6. In that post that provides the fix with `rxjs-compat`, there's also a [Starblitz](https://stackblitz.com/edit/angular-ctllpr?file=package.json) that one of the user made, it runs on Angular 6 but it seems to have lower version of `ngx-translate`, you might want to take a look at it.

There's not much help I can bring since like I said I skipped Angular 6 and most users that had issues with it, eventually upgraded to Angular 7 and got rid of all errors.
